pkg/tfbridge/schema_test.go

```diff
@@ -1695,9 +1695,9 @@ func TestFailureReasonForMissingRequiredFields(t *testing.T) {
 				Schema: &ResourceInfo{
 					Tok: tokens.NewTypeToken("module", "importableResource"),
 					Fields: map[string]*SchemaInfo{
-						"inputY": {
+						"input_y": {
 							Default: &DefaultInfo{
-								Config: "input_y_config",
+								Config: "inputYConfig",
```

Comment:
Here, conversely the framework expects Pulumi PropertyKey notation, though it is not enforced by the test. Looking around Ecosystem maintained bridged providers I did not find any actual uses of this feature, which may explain why this bit-rotted without being detected.